### PR TITLE
adds ability to log km2 for serialtiles

### PR DESCRIPTION
**Pull request by suhasdeshpande**

### CODE REVIEW

File: lib/serialtilescopy.js [hunk 1]
```diff
@@ -67,7 +68,16 @@ function serialtiles(srcUri, s3urlTemplate, options, callback) {
...
             tileSizeStats.max = tile.buffer.length;
-        }
+          }
+          var area = tileSizeStats.hasOwnProperty(tile.z)
+            ? tileSizeStats[tile.z] + utils.calculateTileArea(tile.z, tile.x, tile.y)
+            : utils.calculateTileArea(tile.z, tile.x, tile.y);
+
+          tileSizeStats = {
+            ...tileSizeStats,
+            [z]: area
```
Comment (suhasdeshpande) [inline lib/serialtilescopy.js:78]: [tile.z]: area

Comment (suhasdeshpande) [inline lib/serialtilescopy.js:78]:
```suggestion
            [tile.z]: area
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 7.5.1
+- fix a bug with tile property
+
+## 7.5.0
```
Comment (mapsam) [inline CHANGELOG.md:6]: has 7.5.0 already been published?

Comment (suhasdeshpande) [reply CHANGELOG.md:6]: introduce bug in 7.5.0 so had to release another patch

**Pull request merged**